spool_2/regex_spec.rb

```diff
@@ -4,73 +4,74 @@
 
   let(:phrase) { "It was a bright cold day in April, and the clocks were striking thirteen." }
 
-  it "matches using =~" do
+  it "matches using match" do
```

Comment:
What's the reasoning for changing `=~` to `match`? Is the former deprecated in this Rspec version?